Source/SwiftLintFramework/Rules/TrailingCommaRule.swift

```diff
@@ -99,6 +102,22 @@ public struct TrailingCommaRule: ASTRule, ConfigurationProviderRule {
             )
         ]
     }
+
+    private func trailingCommaIndex(_ contents: String, file: File, offset: Int) -> Int? {
+        // swiftlint:disable:next force_try
+        let regex = try! NSRegularExpression(pattern: ",", options: [.ignoreMetacharacters])
+        let range = NSRange(location: 0, length: contents.characters.count)
+        let ranges = regex.matches(in: contents, options: [], range: range).map { $0.range }
+
+        // skip commas in comments
+        return ranges.filter {
```

Comment:
I don't think this will be an issue since in most cases `ranges` would be empty. We could cache the regex tough.